I'm sorry, but an uncaught exception occurred.

While running game code:
  File "game/cass route.rpy", line 1656, in script
    show h smile w dissolve
Exception: Image 'h smile' does not accept attributes 'w dissolve'.

-- Full Traceback ------------------------------------------------------------

Full traceback:
  File "game/cass route.rpy", line 1656, in script
    show h smile w dissolve
  File "/Applications/renpy-7.4.6-sdk/renpy/ast.py", line 1428, in execute
    show_imspec(self.imspec, atl=getattr(self, "atl", None))
  File "/Applications/renpy-7.4.6-sdk/renpy/ast.py", line 1391, in show_imspec
    renpy.config.show(name,
  File "/Applications/renpy-7.4.6-sdk/renpy/exports.py", line 733, in show
    if not base.find_target() and renpy.config.missing_show:
  File "/Applications/renpy-7.4.6-sdk/renpy/display/image.py", line 421, in find_target
    self.target = target._duplicate(a)
  File "/Applications/renpy-7.4.6-sdk/renpy/display/core.py", line 499, in _duplicate
    args.extraneous()
  File "/Applications/renpy-7.4.6-sdk/renpy/display/core.py", line 362, in extraneous
    raise Exception("Image '{}' does not accept attributes '{}'.".format(
Exception: Image 'h smile' does not accept attributes 'w dissolve'.

macOS-13.2.1-arm64-arm-64bit arm64
Ren'Py 8.1.2.23090503
Crimson Nights 1.0
Wed Oct 25 15:01:18 2023
